app/Lib/Rscripts/analyse.R

```diff
@@ -2,11 +2,45 @@
 library(psy)
 
 Analyse <- function(key, input.answers, number.answeroptions) {
+  # Calculates standard psychometric properties of an exam, notably the 
+  # percentage correct and the item rest correlation (IRC) per item and
+  # per answer option and the cronbach's alpha for the whole exam.
+  #
+  # Arguments:
+  #   key: Matrix of 0's and 1's. key[i,j] implies wether answer option i 
+	#        to item j is right (1) or wrong (0). If a row (item) consists of 
```

Comment:
"(...) When indenting your code, use two spaces.  Never use tabs or mix tabs and spaces. (...)" see: https://google.github.io/styleguide/Rguide.xml#indentation